internal/corazawaf/transaction.go

```diff
@@ -882,7 +882,8 @@ func (tx *Transaction) ReadRequestBodyFrom(r io.Reader) (*types.Interruption, in
 			// bytes.Buffer does not work with this kind of sizes. See comments in BodyBuffer Write(data []byte)
 			return nil, 0, errors.New("Overflow reached while writing request body")
 		}
-		if tx.requestBodyBuffer.length+writingBytes >= tx.RequestBodyLimit {
+		if tx.requestBodyBuffer.length+writingBytes > tx.RequestBodyLimit {
```

Comment:
Why not `>=` I think when you reach the limit you already decide whether reject or process partial? Also, not including the upper limit it is a hell to support readers as in order to know if you superpass the limit you need to read one byte more and by that you already screwed the body.

Reply:
I reverted to `>=`, only kept inside `body_buffer` to effectively write up to the limit.